docset/winserver2022-ps/addsdeployment/Test-ADDSReadOnlyDomainControllerAccountCreation.md

```diff
@@ -16,34 +16,48 @@ Runs the prerequisites for adding a RODC account.
 ## SYNTAX
 
 ```
-Test-ADDSReadOnlyDomainControllerAccountCreation -DomainControllerAccountName <String> -DomainName <String>
- -SiteName <String> [-AllowPasswordReplicationAccountName <String[]>] [-Credential <PSCredential>]
- [-DelegatedAdministratorAccountName <String>] [-DenyPasswordReplicationAccountName <String[]>]
- [-NoGlobalCatalog] [-InstallDns] [-ReplicationSourceDC <String>] [-Force] [<CommonParameters>]
+Test-ADDSReadOnlyDomainControllerAccountCreation -DomainControllerAccountName <String>
+ -DomainName <String> -SiteName <String> [-AllowPasswordReplicationAccountName <String[]>]
+ [-Credential <PSCredential>] [-DelegatedAdministratorAccountName <String>]
+ [-DenyPasswordReplicationAccountName <String[]>] [-NoGlobalCatalog] [-InstallDns]
+ [-ReplicationSourceDC <String>] [-Force] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Test-ADDSReadOnlyDomainControllerAccountCreation** cmdlet runs the prerequisite checks which would be performed if you were to add a read-only domain controller (RODC) account in Active Directory using the Add-ADDSReadOnlyDomainControllerAccount cmdlet.
-It differs from using the *WhatIf* parameter with the **Add-ADDSReadOnlyDomainControllerAccount** cmdlet in that instead of summarizing the changes that would occur during the account creation process, this cmdlet actually tests whether those changes are possible given the current environment.
+
+The `Test-ADDSReadOnlyDomainControllerAccountCreation` cmdlet runs the prerequisite checks which
+would be performed if you were to add a read-only domain controller (RODC) account in Active
+Directory using the Add-ADDSReadOnlyDomainControllerAccount cmdlet. It differs from using the
+`-WhatIf` parameter with the `Add-ADDSReadOnlyDomainControllerAccount` cmdlet in that instead of
+summarizing the changes that would occur during the account creation process, this cmdlet actually
+tests whether those changes are possible given the current environment.
```

Comment:
```suggestion
The `Test-ADDSReadOnlyDomainControllerAccountCreation` cmdlet runs the prerequisite checks which
would be performed if you were to add a read-only domain controller (RODC) account in Active
Directory using the `Add-ADDSReadOnlyDomainControllerAccount` cmdlet. It differs from using the
**WhatIf** parameter with the `Add-ADDSReadOnlyDomainControllerAccount` cmdlet in that instead of
summarizing the changes that would occur during the account creation process, this cmdlet actually
tests whether those changes are possible given the current environment.
```